src/components/Docs/Docs.tsx

```diff
@@ -0,0 +1,23 @@
+import { Link, Text } from '@chakra-ui/react';
+
+type DocsURLProps = {
+  url: string;
+  provider: string;
+  credentialName: string;
+};
+
+export function DocsURL({ url, provider, credentialName }: DocsURLProps) {
```

Comment:
Can we name the component something like `DocsHelperText`, since it does more than just show the URL. You'll have to rename the folder and file name accordingly also.